src/components/Listing/Listing.html

```diff
@@ -0,0 +1,38 @@
+<div class="container">
+  <div class="advertiser-details">
+    <h1 class="title">{{ listing["listing"]["title"] }}</h1>
+    <!-- <span>{{ listing["listing"]["adress"]["locationId"] }}</span> -->
```

Comment:
Is this comment necessary?

Reply:
I was gonna use it later, but already added.